src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -53,35 +53,44 @@
 		</div>
 
 		<div
-			class="item pointer-events-auto flex flex-row items-center justify-end gap-2 text-sm lg:max-w-48 xl:max-w-none"
+			class="item pointer-events-auto px-6 text-sm md:pl-0 md:pr-0 lg:max-w-48"
```

Comment:
Padding can be shortened for md too: md:px-0.

Reply:
@DenysKarmazynDFINITY 
You are right. I just fixed it.